src/components/skills-quiz/SelectJobCard.jsx

```diff
@@ -25,8 +25,9 @@ const SelectJobCard = () => {
 
   return (
     <div>
-      <h3>Your jobs and skills</h3>
+      <h3 id="skills-quiz-jobs-skills-selections">Your jobs and skills</h3>
       <SelectableBox.Set
+        ariaLabelledby="skills-quiz-jobs-skills-selections"
```

Comment:
[inform] Breaking change for Paragon v21 upgrade; requires either the `ariaLabelledby` or `ariaLabel` prop.